source/frontend/src/features/mapSideBar/layout/MapSideBarLayout.tsx

```diff
@@ -115,6 +115,9 @@ const StyledSidebarWrapper = styled.div`
 
   &.expanded {
     min-width: 90rem;
+    @media only screen and (max-width: 1199px) {
+      min-width: 70rem;
```

Comment:
same comment as above, this seemed like a good balance between maintaining map usability and reserving enough space on the left side for functional UI.